### PR TITLE
Update tatoeba to v2021-07-22

### PR DESCRIPTION
**Pull request by KoichiYasuoka**
Tatoeba's latest version is v2021-07-22

### CODE REVIEW

**Comment by KoichiYasuoka** [comment]
How about this? @lhoestq @abhishekkrthakur

**Comment by lhoestq** [comment]
Hi ! I think it would be nice if people could still be able to load the old version.
Maybe this can be a parameter ? For example to load the old version they could do
```python
load_dataset("tatoeba", lang1="en", lang2="mr", date="v2020-11-09")
```

If it sounds good to you, we can add this parameter to the TatoebaConfig:
```python
class TatoebaConfig(datasets.BuilderConfig):
    def __init__(self, *args, lang1=None, lang2=None, date="v2021-07-22", **kwargs):
    self.date = date
```
and then pass the date to the URL
```python
_BASE_URL = "https://object.pouta.csc.fi/OPUS-Tatoeba/{}/moses/{}-{}.txt.zip"
```
```python
        def _base_url(lang1, lang2, date):
            return _BASE_URL.format(date, lang1, lang2)
```

What do you think ?

**Comment by KoichiYasuoka** [comment]
`_DATE = "v" + "-".join(s.zfill(2) for s in _VERSION.split("."))` seems rather tricky but works well. How about this? @lhoestq

File: datasets/tatoeba/tatoeba.py [hunk 1]
```diff
@@ -95,10 +97,10 @@ def _info(self):
         )
 
     def _split_generators(self, dl_manager):
-        def _base_url(lang1, lang2):
-            return _BASE_URL.format(lang1, lang2)
+        def _base_url(lang1, lang2, date):
+            return.format(date, lang1, lang2)
```
Comment (lhoestq) [inline datasets/tatoeba/tatoeba.py:101]:
```suggestion
            return _BASE_URL.format(date, lang1, lang2)
```

**Review by lhoestq — APPROVED** [review]
Looks all good thanks !

I just renamed the parent directory of the dummy data to match the new version name, and mentioned how to change the date in the dataset card.

**Comment by lhoestq** [comment]
The CI is only failing because of the missing sections in the dataset card, and because of an issue with the CER metric that is unrelated to this PR

**Pull request merged**